### ISSUE TITLE
Hosting on Firebase Hosting

### ISSUE THREAD

**Issue by RajarsiGit** [SandipanDGit/Collaborative-classroom-quiz#1]
Hello @SandipanDGit it's a great project and has been implemented using beautiful ideas. Since this is a single page application and moreover the entire logic is in the frontend JavaScript. I would suggest you try hosting this on Google Firebase Hosting.
This will not only let you visualise your application on a public domain but also would let others quickly have a look at your work without having to have to clone your repository to run your application.
Thanks.

**Comment by SandipanDGit** [comment]
@RajarsiGit Yes you are right, I should host it. I have thought about it but haven't zeroed down on which server to deploy to, GitHub pages, firebase or netlify. I am still working on it and soon it will be up for live demo I hope.
